healthtools/setupLogging.py

```diff
@@ -0,0 +1,12 @@
+import logging
+import logging.config
+from healthtools.config import LOGGING
+
+def setup_logging(default_level=logging.INFO):
+    """
+    Setup logging configuration
+    """
+    try:
+        logging.config.dictConfig(LOGGING)
+    except Exception as ex:
+        logging.basicConfig(level=default_level)
```

Comment:
Really small file. Could we have it in `scraper.py` instead?

Reply:
ok, I would implement the feedback. Thanks